src/lib/components/ExploreCard.svelte

```diff
@@ -39,8 +40,11 @@
     <Button
       variant="outline"
       icon={Apply}
-      disabled={$walletStore.connected &&
-        workstream.applicants?.includes($walletStore.address)}
+      disabled={!(
+        $connectedAndLoggedIn &&
```

Comment:
Could we not store this whole check in the store. I feel we repeat this in a few places